source/synthDriverHandler.py

```diff
@@ -409,10 +407,10 @@ def _getSynthDriver(name) -> SynthDriver:
 	return importlib.import_module("synthDrivers.%s" % name, package="synthDrivers").SynthDriver
 
 
-def getSynthList() -> List[Tuple[str, str]]:
+def getSynthList() -> list[SynthDriver]:

```

Comment:
please don't change the return type of this function, it is an API breaking change